synapseclient/client.py

```diff
@@ -2011,23 +2011,25 @@ def getTeam(self, id):
         """
         Finds a team with a given ID or name.
 
-        :param id:  The ID or name of the team to retrieve
+        :param id:  The ID or name of the team or a Team object to retrieve
```

Comment:
what is the use case to extend the API to take a team object?

Reply:
I was just updating the docstring, because the function does
``` 
teamid = id_of(id)
```